### PR TITLE
made changes/fixes to tour flow

### PR DESCRIPTION
**Pull request by heyitsmebev**
This PR focuses on cleaning up issues/bugs in the tour flow. 
It targets styling, backlink, and template layout

Ticket: https://github.com/GSA/notifications-admin/issues/1256

> The styling was fixed when the new `withnav_template.html` was merged. However, the tour flow was a bit odd. When the user went through the tour flow, it would ask the user to select a time when there were no input fields, and the headers were not correctly describing the current step in the tour flow. In addition, the back link was not properly linking to the correct area. I fixed the tour flow by isolating our send and tour flow so that the tour flow would look more complete and unified. Originally, it was inheriting the changes made to the send flow.
> 
> ### **The old verison**
image
> 
> ### **New Version**
image

### CODE REVIEW

File: tests/end_to_end/test_send_message_from_existing_template.py [hunk 1]
```diff
@@ -256,13 +256,6 @@ def handle_existing_template_case(page):
     # Check to make sure that we've arrived at the next page.
     page.wait_for_load_state("domcontentloaded")
 
-    preview_button = page.get_by_text("Preview")
```
Comment (alexjanousekGSA) [inline tests/end_to_end/test_send_message_from_existing_template.py:259]: I am not super familiar with this current flow. Does this button not exist anymore or is the test being odd and not recognizing the button?

**Review by alexjanousekGSA — APPROVED** [review]

File: tests/end_to_end/test_send_message_from_existing_template.py (hunk 1 above)
Comment (ccostino) [reply tests/end_to_end/test_send_message_from_existing_template.py:259]: If I'm understanding the changes here correctly, the template used for this page changed and therefore this is an intermediary step/page that is no longer needed for this flow.

However, I didn't think we were getting rid of the message preview step in the actual message send flow?  That's what this test is for, so I wonder if the changes in this PR have effects elsewhere in the full send message flow.

Comment (heyitsmebev) [reply tests/end_to_end/test_send_message_from_existing_template.py:259]: Ah yes, makes sense. Thank you for pointing that out. I made an adjustment for the tour vs regular send flow @ccostino @alexjanousekGSA

**Review by ccostino — APPROVED** [review]
Thanks, @heyitsmebev!

**Pull request merged**